src/libcore/iter.rs

```diff
@@ -2448,6 +2459,8 @@ impl<I> Fuse<I> {
     /// previously returned `None`.
     #[inline]
     #[unstable(feature = "iter_reset_fuse", reason = "seems marginal")]
+    #[deprecated(since = "1.3.0",
+                 reason = "unusual for adaptors to have one-off methods")]
```

Comment:
This is an unusual comment :-)  I don't disagree with the deprecation, but I'd like to add a method to fuse, to query its fused/nonfused state, it would be quite useful.